src/ExtractToFStar.ml

```diff
@@ -1471,6 +1477,108 @@ let extract_fun_decl (ctx : extraction_ctx) (fmt : F.formatter)
   (* Add breaks to insert new lines between definitions *)
   F.pp_print_break fmt 0 0
 
+(* Change the suffix from "_c" to "_body" *)
+let global_decl_to_body_name (decl : string) : string =
+  (* The declaration length without the global suffix *)
+  let base_len = String.length decl - 2 in
+  (* TODO: Use String.ends_with instead when a more recent version of OCaml is used *)
+  assert (String.sub decl base_len 2 = "_c");
+  (String.sub decl 0 base_len) ^ "_body"
+
+(** Print a definition of the shape "QUALIF NAME : TYPE = BODY" with a custom body extractor *)
+let extract_global_definition (ctx : extraction_ctx) (fmt : F.formatter)
```

Comment:
Ty ! For the renaming, i'm not sure since it also handle the signature part, as for me definition = declaration/signature + body/implementation (we extract both here).

Reply:
I renamed it `extract_global_decl_body` then ^^